ci/Dockerfile

```diff
@@ -5,6 +5,12 @@ ARG GINKGO_VERSION=latest
 ARG GOLANGCILINT_VERSION=latest
 RUN apt-get update && apt-get install -y libpcap-dev && rm -rf /var/lib/apt/lists/*
 
+RUN curl -fsSL https://deb.nodesource.com/setup_current.x | sudo -E bash - && \
+    apt-get install -y nodejs && rm -rf /var/lib/apt/lists/*
+
+RUN npm install -g semantic-release && \
+    npm install -g @semantic-release/exec
```

Comment:
This could be in the line above. NPM should be able to install different packages at once.

Reply:
i'd rather have each package on its own line for clarity. yes we can install two packaged in one command though.